### DISCUSSION TITLE
Add set possibility to brackets

### DISCUSSION THREAD

**Discussion by mknies** [evroon/bracket#874] [Ideas]
Would it be possible to add a set possibility to brackets because many sports use it (tennis, table-tennis, volleyball etc.)?

**Comment by evroon** [comment]
Yes I was also thinking about this, it will increase complexity which I am trying to avoid.
However, I play badminton myself, so I can see why it's very valuable.
So I'll make it an [issue](https://github.com/evroon/bracket/issues/877)

**Reply by mknies** [reply]
Great, thank you soooo much 😃
